### PR TITLE
Add cmds into wsctl

### LINKED ISSUE #245: command to create/delete publisher

### PR DESCRIPTION
**Pull request by Liuhaai**
fix #245 #244  #243

### CODE REVIEW

File: cmd/srv-applet-mgr/apis/applet/del.go [hunk 1]
```diff
@@ -13,11 +13,15 @@ type RemoveApplet struct {
...
+func (r *RemoveApplet) Path() string { return "/:appletID" }
 
 func (r *RemoveApplet) Output(ctx context.Context) (interface{}, error) {
 	a := middleware.CurrentAccountFromContext(ctx)
-	if _, err := a.ValidateProjectPerm(ctx, r.ProjectID); err != nil {
+	app, err := applet.GetAppletByAppletID(ctx, r.AppletID)
+	if err != nil {
+		return nil, err
+	}
+	if _, err := a.ValidateProjectPerm(ctx, app.ProjectID); err != nil {
```
Comment (saitofun) [inline cmd/srv-applet-mgr/apis/applet/del.go:24]: currentuser.WithProjectContext is recommended
it validates and add dependencies to context

File: pkg/modules/applet/applet.go [hunk 2]
```diff
@@ -224,8 +224,7 @@ func ListApplets(ctx context.Context, r *ListAppletReq) (*ListAppletRsp, error)
 }
 
 type RemoveAppletReq struct {
-	ProjectID types.SFID `in:"path"  name:"projectID"`
-	AppletID  types.SFID `in:"path"  name:"appletID"`
+	AppletID types.SFID `in:"path"  name:"appletID"`
 }
 
 func RemoveApplet(ctx context.Context, r *RemoveAppletReq) error {
```
Comment (saitofun) [inline pkg/modules/applet/applet.go:230]: this struct can be removed

File: pkg/wsctl/client/client.go [hunk 3]
```diff
@@ -71,24 +73,23 @@ func (c *client) SelectTranslation(trls map[config.Language]string) string {
...
+		return nil, err
 	}
-	if cr.Code/1e6 == 401 {
-		c.login()
-		return c.call(url, req, c.getToken())
+	if gjson.ValidBytes(body) {
+		ret := gjson.ParseBytes(body)
+		if code := ret.Get("code"); code.Exists() && (code.Uint()/1e6 == 401) {
+			c.login()
+			return c.Call(url, req)
```
Comment (hunshenshi) [inline pkg/wsctl/client/client.go:89]: Is a threshold needed to avoid endless loops

File: cmd/srv-applet-mgr/apis/applet/del.go (hunk 1 above)
Comment (Liuhaai) [reply cmd/srv-applet-mgr/apis/applet/del.go:24]: It could be done in a separated pr.

File: pkg/modules/applet/applet.go (hunk 2 above)
Comment (Liuhaai) [reply pkg/modules/applet/applet.go:230]: It could be done in a separated pr.

**Pull request merged**